src/Symfony/Component/Translation/Extractor/Visitor/TranslatableMessageVisitor.php

```diff
@@ -39,7 +40,7 @@ public function leaveNode(Node $node): ?Node
             return null;
         }
 
-        if (!\in_array('TranslatableMessage', $className->getParts(), true)) {
+        if (TranslatableMessage::class !== $className->name) {
```

Comment:
`is_a` so it can work with child classes ?

Reply:
Child classes can change the constructor, so this would not be safe to assume the meaning of arguments.